**Рост взрослого населения города X имеет нормальное распределение. Причем, средний рост равен 174 см, а среднее квадратичное отклонение равно 8 см. Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:**  

               а) больше 182 см  
               б) больше 190 см  
               в) от 166 см до 190 см  
               г) от 166 см до 182 см  
               д) от 158 см до 190 см  
               е) не выше 150 см или не ниже 190 см  
               ё) не выше 150 см или не ниже 198 см  
               ж) ниже 166 см.  

Для расчета значений вероятности воспользуемся методом cdf, объекта NormalDist стандартного модуля statistics 

In [51]:
from statistics import NormalDist

Напишем функцию для определения $Z$-значений для данного распределения:

In [52]:
def z_value(height):
    return (height-174)/8

a) Определим $Z$-значениe:

In [53]:
print(z_value(182))

1.0


метод cdf определяет вероятность того, что случайная величина X будет меньше или равна x, т.е в данном случае $P(X\le182)$ 

In [54]:
NormalDist().cdf(z_value(182))

0.8413447460685428

Однако нам нужна вероятность $P(X>182)$, поэтому вычтем полученное значение из $1$:

In [55]:
P=1-NormalDist().cdf(z_value(182))
print(f'Вероятность того, что рост случайно выбранного человека будет больше 182 см составляет {P: .4f}')

Вероятность того, что рост случайно выбранного человека будет больше 182 см составляет  0.1587


Вероятность того, что рост случайно выбранного человека будет больше 182 см составляет - $P(X>182)=0.1587$ или $\approx15,87\%$  

---

б) Определим $Z$-значениe:

In [56]:
z=z_value(190)
z

2.0

Определим $P(X>190)$

In [57]:
P=1-NormalDist().cdf(z)
print(f'Вероятность того, что рост случайно выбранного человека будет больше 190 см составляет {P: .4f}')

Вероятность того, что рост случайно выбранного человека будет больше 190 см составляет  0.0228


Вероятность того, что рост случайно выбранного человека будет больше 190 см составляет - $P(X>190)=0.0228$ или $\approx2,28\%$  

---

в) Определим $Z_1$ и $Z_2$

In [58]:
z1=z_value(166)
z1

-1.0

In [59]:
z2=z_value(190)
z2

2.0

Определим $P(166\le X\le 190)$

In [60]:
P=NormalDist().cdf(z2)-NormalDist().cdf(z1)
print(f'Вероятность того, что рост случайно выбранного человека будет от 166 см до 190 см составляет {P: .4f}')

Вероятность того, что рост случайно выбранного человека будет от 166 см до 190 см составляет  0.8186


Вероятность того, что рост случайно выбранного человека будет от 166 см до 190 см составляет - $P(166\le X\le 190)=0.8186$ или $\approx81,86\%$  

---

г) Определим $Z_1$ и $Z_2$

In [61]:
z1=z_value(166)
z1

-1.0

In [62]:
z2=z_value(182)
z2

1.0

Определим $P(166\le X\le 182)$

In [63]:
P=NormalDist().cdf(z2)-NormalDist().cdf(z1)
print(f'Вероятность того, что рост случайно выбранного человека будет от 166 см до 182 см составляет {P: .4f}')

Вероятность того, что рост случайно выбранного человека будет от 166 см до 182 см составляет  0.6827


Вероятность того, что рост случайно выбранного человека будет от 166 см до 182 см составляет - $P(166\le X\le 182)=0.6827$ или $\approx68,27\%$  

---

д) Определим $Z_1$ и $Z_2$

In [64]:
z1=z_value(158)
z1

-2.0

In [65]:
z2=z_value(190)
z2

2.0

Определим $P(158\le X\le 190)$

In [66]:
P=NormalDist().cdf(z2)-NormalDist().cdf(z1)
print(f'Вероятность того, что рост случайно выбранного человека будет от 158 см до 190 см составляет {P: .4f}')

Вероятность того, что рост случайно выбранного человека будет от 158 см до 190 см составляет  0.9545


Вероятность того, что рост случайно выбранного человека будет от 158 см до 190 см составляет - $P(158\le X\le 190)=0.9545$ или $\approx95,45\%$  

---

е) Определим $Z_1$ и $Z_2$

In [67]:
z1=z_value(150)
z1

-3.0

Так как метод cdf определяет вероятность того, что случайная величина X будет меньше или равна x, т.е для расчета вероятности роста человека **"не ниже"** необходимо применять значение на 0,1 меньше _(при измерении роста с точностью до миллиметра)_ - $P(X\le189.9)$ 

In [68]:
z2=z_value(189.9)
z2

1.9875000000000007

Определим $P(X\le150\cup X\ge 190)$

In [69]:
P=NormalDist().cdf(z1)+(1-NormalDist().cdf(z2))
print(f'Вероятность того, что рост случайно выбранного человека будет не выше 150 см или не ниже 190 см составляет {P: .4f}')

Вероятность того, что рост случайно выбранного человека будет не выше 150 см или не ниже 190 см составляет  0.0248


Вероятность того, что рост случайно выбранного человека будет не выше 150 см или не ниже 190 см составляет $P(X\le150\cup X\ge 190)=0.0248$ или $\approx2,48\%$    

---

ё) Определим $Z_1$ и $Z_2$

In [70]:
z1=z_value(150)
z1

-3.0

In [71]:
z2=z_value(197.9)
z2

2.9875000000000007

Определим $P(X\le150\cup X\ge 198)$

In [72]:
P=NormalDist().cdf(z1)+(1-NormalDist().cdf(z2))
print(f'Вероятность того, что рост случайно выбранного человека будет не выше 150 см или не ниже 198 см составляет {P: .4f}')

Вероятность того, что рост случайно выбранного человека будет не выше 150 см или не ниже 198 см составляет  0.0028


Вероятность того, что рост случайно выбранного человека будет не выше 150 см или не ниже 198 см составляет $P(X\le150\cup X\ge 198)=0.0028$ или $\approx0,28\%$    

---

ж) Определим $Z$-значениe:

In [73]:
z=z_value(165.9)
z

-1.0124999999999993

Определим $P(X<166)$

In [74]:
P=NormalDist().cdf(z)
print(f'Вероятность того, что рост случайно выбранного человека ниже 166 см составляет {P: .4f}')

Вероятность того, что рост случайно выбранного человека ниже 166 см составляет  0.1556


Вероятность того, что рост случайно выбранного человека будет ниже 166 см составляет $P(X<166)=0.1556$ или $\approx15,56\%$  

---

Ответ. Вероятность того, что рост случайно выбранного человека будет:  
а) больше 182 см - $P(X>182)=0.1587$ или $\approx15,87\%$  
б) больше 190 см - $P(X>190)=0.0228$ или $\approx2,28\%$  
в) от 166 см до 190 см - $P(166\le X\le 190)=0.8186$ или $\approx81,86\%$  
г) от 166 см до 182 см - $P(166\le X\le 182)=0.6827$ или $\approx68,27\%$  
д) от 158 см до 190 см - $P(158\le X\le 190)=0.9545$ или $\approx95,45\%$  
е) не выше 150 см или не ниже 190 см - $P(X\le150\cup X\ge 190)=0.0248$ или $\approx2,48\%$  
ё) не выше 150 см или не ниже 198 см - $P(X\le150\cup X\ge 198)=0.0028$ или $\approx0,28\%$  
ж) ниже 166 см - $P(X<166)=0.1556$ или $\approx15,56\%$  

---